### DISCUSSION TITLE
Resolve current index error

### DISCUSSION THREAD

**Discussion by slferris** [ohmyzsh/ohmyzsh#12080] [Help] [answered]
With command "omz update", I get a message "themes/robbyrussell.zsh-theme: needs merge error: you need to resolve your current index first". How do I resolve my current index and correct "merge error"?

**Comment by mcornella** [comment, marked as answer]
It seems the file has some changes, you can remove all changes to fix the error, with the steps below:
```
cd $ZSH
git reset --hard HEAD
```

Note that this will remove all changes if you made any, so you'd want to save them before doing that if you care about them. If you are making any changes to the theme, you can review [how to customize the theme](https://github.com/ohmyzsh/ohmyzsh/wiki/Customization#overriding-and-adding-themes) which explains how to do it properly so that there are no conflicts when updating.
